packages/overlays/src/OverlayController.js

```diff
@@ -102,6 +102,7 @@ export class OverlayController extends EventTargetShim {
       contentWrapperNode: config.contentWrapperNode,
       invokerNode: config.invokerNode,
       backdropNode: config.backdropNode,
+      hasBackdropAnimation: false,
```

Comment:
Could we do animation in transitionHide/transitonShow, just like with the contentNode? I think it's a Deisgn System implementation detail.
And then we are consistent: all animation happen in the Subclasser layer. We only provide proper timing alignments by exposing promise hooks